addons/account/models/account_reconcile_model.py

```diff
@@ -884,7 +884,7 @@ def _check_rule_propositions(self, statement_line, candidates):
         # Statement line amount is equal to the total residual.
         if line_currency.is_zero(line_residual_after_reconciliation):
             return True
-        reconciled_percentage = 100 - abs(line_residual_after_reconciliation) / abs(line_residual - line_residual_after_reconciliation) * 100
+        reconciled_percentage = 100 - abs(line_residual_after_reconciliation) / abs(line_residual - line_residual_after_reconciliation) * 100 if (line_residual - line_residual_after_reconciliation != 0) else 0
```

Comment:
Could you please make the line shorter by either
* using a `if` clause instead of a ternary operation
* using a variable for the divisor

Reply:
Done, thanks !